spec/forms/candidate_interface/personal_details_form_spec.rb

```diff
@@ -86,57 +62,7 @@
     it { is_expected.to validate_length_of(:last_name).is_at_most(60) }
 
     describe 'date of birth' do
-      around do |example|
-        Timecop.freeze(Time.zone.local(2019, 1, 1)) do
-          example.run
-        end
-      end
-
-      it 'is invalid if not well-formed' do
-        personal_details = CandidateInterface::PersonalDetailsForm.new(
-          day: '99', month: '99', year: '99',
-        )
-
-        personal_details.validate
-
-        expect(personal_details.errors.full_messages_for(:date_of_birth)).to eq(
-          ['Date of birth Enter a real date of birth'],
-        )
-      end
-
-      it 'is invalid if the date is in the future' do
-        personal_details = CandidateInterface::PersonalDetailsForm.new(
-          day: '2', month: '8', year: '2999',
-        )
-
-        personal_details.validate
-
-        expect(personal_details.errors.full_messages_for(:date_of_birth)).to eq(
-          ['Date of birth Enter a date of birth that is in the past, for example 31 3 1980'],
-        )
-      end
-
-      it 'is invalid if the candidate is younger than 16' do
-        personal_details = CandidateInterface::PersonalDetailsForm.new(
-          day: '2', month: '1', year: '2003',
-        )
-
-        personal_details.validate
-
-        expect(personal_details.errors.full_messages_for(:date_of_birth)).to eq(
-          ['Date of birth Enter a date of birth before 1 January 2003 – you must be over 16 years old to Apply for teacher training'],
-        )
-      end
-
-      it 'is valid if the candidate is older than 16' do
-        personal_details = CandidateInterface::PersonalDetailsForm.new(
-          day: '31', month: '12', year: '2002',
-        )
-
-        personal_details.validate
-
-        expect(personal_details.errors.full_messages_for(:date_of_birth)).to be_empty
-      end
+      include_examples 'date_of_birth validations', verify_presence: true
```

Comment:
Merging this to fix the bug, but I think we should probably keep this inlined if it's only used in one place?

Reply:
@duncanjbrown the `date_of_birth` is also used in the edit forms so the plan is to reuse these validations there and tidy up the current spec!